lib/active_record/virtual_attributes/virtual_total.rb

```diff
@@ -151,4 +151,4 @@ def virtual_aggregate_arel(reflection, method_name, column)
   end
 end
 
-ActiveRecord::Base.send(:include, VirtualAttributes::VirtualTotal)
+ActiveRecord::Base.include VirtualAttributes::VirtualTotal
```

Comment:
Weird, why did we use send before?

Reply:
A pervious version of ruby made `include` private. Sorry, can't remember the exact version.
Now that we are only supporting more recent versions of ruby, we can use the standard `include`

Reply:
Ah, ok, thanks!